packages/wonder-blocks-dropdown/components/check-icons.js

```diff
@@ -0,0 +1,131 @@
+// @flow
+
+/**
+ * Check and checkbox components used by option item.
+ */
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import Color, {mix} from "@khanacademy/wonder-blocks-color";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Icon, {icons} from "@khanacademy/wonder-blocks-icon";
+
+import type {IconAsset} from "@khanacademy/wonder-blocks-icon";
+
+const {
+    blue,
+    white,
+    offBlack,
+    offBlack16,
+    offBlack32,
+    offBlack50,
+    offWhite,
+} = Color;
+
+type CheckProps = {|
+    disabled: boolean,
```

Comment:
Would be good to add explicit comments to each of these to explain them `/** ... */`